common/src/main/scala/models/Registration.scala

```diff
@@ -1,6 +1,6 @@
 package models
 
-case class Registration(deviceId: String, platform: Platform, userId: UserId, topics: Set[Topic])
+case class Registration(deviceId: String, platform: Platform, userId: UniqueDeviceIdentifier, topics: Set[Topic])
```

Comment:
I know, but its best time now to deprecate old name now at the same time introducing new since we don't have other platforms yet.